src/scripts/background.js

```diff
@@ -463,7 +464,7 @@ window.TogglButton = {
       pid: timeEntry.pid || timeEntry.projectId || null,
       tid: timeEntry.tid || null,
       wid: timeEntry.wid || TogglButton.$user.default_wid,
-      tags: timeEntry.tags || null,
+      tags: enableAutoTagging ? timeEntry.tags || null : null,
```

Comment:
Not sure if it is related with the linter 🤷‍♂️ 
```suggestion
      tags: enableAutoTagging ? (timeEntry.tags || null) : null,
```
or
```suggestion
      tags: (enableAutoTagging && timeEntry.tags) || null
```